### ISSUE TITLE
UnnecessaryFullyQualifiedName false positive with PascalCase fields

### ISSUE THREAD

**Issue by Gama11** [detekt/detekt#9018]
```kotlin
class Foo {
    val PascalCase: String = "test"
}

fun main() {
    val foo = Foo()
    println(foo.PascalCase.substring(1))
}
```

>Fully qualified static method call 'foo.PascalCase.substring' can be replaced with an import. [UnnecessaryFullyQualifiedName]

**Comment by eygraber** [comment]
I'm seeing this a lot with Compose functions in a class:

```kotlin
class Foo {
  @Composable
  fun BarEffect() {}
}

val foo = Foo()
foo.BarEffect()
```
